src/__stories__/boxed-story.tsx

```diff
@@ -1,42 +1,32 @@
 import * as React from 'react';
-import {Box, Boxed, Stack, Text8, skinVars} from '..';
-import {ThemeVariant} from '../theme-variant-context';
-import {useCheckbox} from './helpers';
+import {Box, Boxed, Text8, ResponsiveLayout} from '..';
 
 export default {
     title: 'Components/Primitives/Boxed',
+    parameters: {fullScreen: true},
```

Comment:
this is removing the default story padding of 16px, but then you're manually adding it with a `<Box padding={16}>`

Is there any reason for this?

Reply:
No screenshot changes for this story?